### PR TITLE
Task responsive

### PR DESCRIPTION
**Pull request by C0deJack**
This pull is for:
updating styling 
adding responsive styling 
addition of images and content

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -20,13 +20,22 @@ h1 {
...
+    height: 450px;
 }
 
 nav {
     position: relative;
-    height: 350px;
+    height: 365px;
 }
 
 [class|=nav]  {
```
Comment (JamesCullen2) [inline styles.css:41]: what is this function? what is it doing?

File: styles.css [hunk 2]
```diff
@@ -88,84 +98,191 @@ h3 {
...
+    width: 100%;
     overflow: auto;
-  padding-top: 60px;
 }
 
-main div {
-    width: 1600px;
+.portfolio-box {
+    width: 1700px;
     overflow: auto;
```
Comment (JamesCullen2) [inline styles.css:158]: do you need overflow:auto in both main selector and .portfolio-box selector ? could these be combined?

File: styles.css [hunk 3]
```diff
@@ -88,84 +98,191 @@ h3 {
...
-    margin: 10px;
+    margin-bottom: 65px;
 }
 
-article div {
-    width: 410px;
+.img-holder {
+    width: 390px;
     height: 260px;
     margin: 0 auto;
```
Comment (JamesCullen2) [inline styles.css:173]: could you combine margin:auto for several selectors in one?

File: styles.css [hunk 4]
```diff
@@ -88,84 +98,191 @@ h3 {
...
+    color: #ABB2B9;
 }
 
 /*=====================RESPONSIVE========================*/
+
+@media screen and (max-width: 1400px){
+
+    .portfolio-box {
+        width: 950px;
+    }
```
Comment (JamesCullen2) [inline styles.css]: add line break to stay consistent with line breaks below

**Review by JamesCullen2 — COMMENTED** [review]
V clean code, only a few minor suggestions / queries. J

File: styles.css (hunk 2 above)
Comment (C0deJack) [reply styles.css:158]: you're right it wasn't needed in the main tag so i've removed it.

File: styles.css (hunk 3 above)
Comment (C0deJack) [reply styles.css:173]: maybe, but i've tried to keep the css in separate sections for easy navigation so i might leave them as they are.

File: styles.css (hunk 4 above)
Comment (C0deJack) [reply styles.css]: done. Thanks

File: styles.css (hunk 1 above)
Comment (C0deJack) [reply styles.css:41]: Selecting any of the classes with nav in part of the class name.

**Review by JamesCullen2 — APPROVED** [review]

**Pull request merged**